include/nuttx/net/usrsock.h

```diff
@@ -96,8 +96,8 @@ enum usrsock_message_types_e
 
 begin_packed_struct struct usrsock_request_common_s
 {
-  int8_t reqid;
-  uint8_t xid;
+  uint64_t xid;
+  int32_t  reqid;
```

Comment:
int16_t  reqid?

Reply:
restore the reqid type to int8_t, and add reserved byte to keep the aligned access